src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBase.cpp

```diff
@@ -149,6 +149,32 @@ void FlowSolverBase::registerDataOnMesh( Group & meshBodies )
   }
 }
 
+real64 FlowSolverBase::solverStep( real64 const & time_n,
```

Comment:
is this `solverStep` function even necessary? To me it looks like it's doing the same as `SolverBase`.

Reply:
Yes, for now it is necessary because the `solverStep` function is not implemented in `SolverBase`, but you are right that this function is a complete duplication of `solverStep` in `ContactSolverBase.cpp`, and also a duplication of `fullyCoupledSolverStep` in `CoupledSolver.hpp`. 

If we are ok to move the implementation of `solverStep` to `SolverBase.cpp`, then we get rid of these functions. And potentially, in `CoupledSolver.hpp`, we could just have:
```
virtual real64 fullyCoupledSolverStep( real64 const & time_n,                                                                                                                          
                                         real64 const & dt,                                                                                                                              
                                         int const cycleNumber,                                                                                                                          
                                         DomainPartition & domain )                                                                                                                      
 {       
      SolverBase::solverStep( time_n, dt, cycleNumber, domain );
 }
```
What do you think?